src/main/java/core/basesyntax/FruitTransactionImpl.java

```diff
@@ -0,0 +1,37 @@
+package core.basesyntax;
+
+import core.basesyntax.service.StorageActionsService;
+import core.basesyntax.strategy.OperationStrategy;
+import java.util.List;
+
+public class FruitTransactionImpl implements FruitTransaction {
+    private static final String PROCEDURE = "Addition";
+    private static final String SEPARATOR = ",";
+    private static final int FRUIT_TYPE_INDEX = 1;
+    private static final int QUANTITY_INDEX = 2;
+    private final OperationStrategy strategy;
+    private final StorageActionsService storageActions;
+
+    FruitTransactionImpl(OperationStrategy strategy, StorageActionsService storageActions) {
+        this.strategy = strategy;
+        this.storageActions = storageActions;
+    }
```

Comment:
you need to write your services in such a way that they return some temporary  data and pass this data to the next service 
<img width="1081" alt="image" src="https://user-images.githubusercontent.com/96740815/197971171-68766c9c-a4f6-4c31-9c39-4c67acc4ad0a.png">